### PR TITLE
OCPBUGS-6590: Address CVE-2022-41717

### PR DESCRIPTION
**Pull request by pierreprinetti**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -10,7 +10,7 @@ require (
 	github.com/onsi/gomega v1.10.1
 	github.com/pborman/uuid v1.2.0
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/net v0.0.0-20220906165146-f3363e06e74c
+	golang.org/x/net v0.4.0
```
Comment (MaysaMacedo) [inline go.mod:13]: Should it be 0.4.0 or 0.5.0 as done on the csi-driver-manila?
https://github.com/openshift/csi-driver-manila-operator/pull/165/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R87

Comment (pierreprinetti) [reply go.mod:13]: [v0.4.0 is the minimum version that contains the fix](https://pkg.go.dev/vuln/GO-2022-1144).

**Pull request merged**